pombola/south_africa/templates/south_africa/mp_attendance.html

```diff
@@ -0,0 +1,112 @@
+{% extends 'base.html' %}
+{% load compressed static %}
+
+{% block title %}MP Attendance{% endblock %}
+
+{% block extra_js_to_load %}
+    addExtraJS([ {% compressed_js 'attendance-table' %} null ]);
+{% endblock %}
+
+{% block css_headers %}
+  {{ block.super }}
+  <link rel="stylesheet" type="text/css" href="{% static 'css/libs/datatables.min.css' %}" />
+{% endblock %}
+
+{% block body_attributes %} class="mp-attendance" {% endblock %}
+
+{% block content %}
+<h1 class="page-title">MP Attendance</h1>
+<div class="major-column">
+  <p>Explore the commitee meeting attendance records for the members of parliament.</p>
+</div>
+
+<div class='mp-attendance-filter'>
+  <div class="col-50">
+    <form class='select-year' action="{% url 'mp-attendance' %}" method="get">
+      {% csrf_token %}
```

Comment:
The `{% csrf_token %}` is unnecessary since all the requests are made with `GET` now, and leaving this in means you end up with the CSRF token in the URL when changing the view of the data.